src/puppetlabs/puppetdb/query.clj

```diff
@@ -65,7 +65,7 @@
             [puppetlabs.puppetdb.jdbc :as jdbc]
             [puppetlabs.puppetdb.time :refer [to-timestamp]]
             [puppetlabs.kitchensink.core :refer [parse-number keyset valset order-by-expr?]]
-            [puppetlabs.puppetdb.scf.storage-utils :refer [db-serialize sql-as-numeric sql-array-query-string sql-regexp-match sql-regexp-array-match]]
+            [puppetlabs.puppetdb.scf.storage-utils :as sutils :refer [db-serialize sql-as-numeric sql-array-query-string sql-regexp-match sql-regexp-array-match]]
```

Comment:
FWIW, I sometimes prefer refers (in some cases I think it can make the code (formatting) easier to read), but it depends on the situation.  In any case, I'd definitely prefer wrapping the long lines if we do keep the refers.